### ISSUE TITLE
[Pre-Spork] Change EVM related execution effort weight

### ISSUE THREAD

**Issue by janezpodhostnik** [onflow/service-account#314]
# Context

Currently the execution effort weight for EVM gas is 0. This is ok at the moment as there is the EVM is not deployed to mainnet yet.

There is a separate issue (#315) to change the execution effort weights and cost after the spork, but the EVM gas weight has to be done before the spork, otherwise in the window after the spork and before the multisign event there is a possibility to exploit the 0 cost EVM gas.

for more context see #315

# Transaction

- **Body**: https://github.com/onflow/flow-core-contracts/blob/1655a34a1ff5b9a3272b7bcdd791101bd0af1e52/transactions/FlowServiceAccount/set_execution_effort_weights.cdc
- **Signer**: service account
- **Arguments**: 
```json
[
  {
    "type": "Dictionary",
    "value": [
      {
        "key": {
          "type": "UInt64",
          "value": "1001"
        },
        "value": {
          "type": "UInt64",
          "value": "1569"
        }
      },
      {
        "key": {
          "type": "UInt64",
          "value": "1002"
        },
        "value": {
          "type": "UInt64",
          "value": "1569"
        }
      },
      {
        "key": {
          "type": "UInt64",
          "value": "1003"
        },
        "value": {
          "type": "UInt64",
          "value": "1569"
        }
      },
      {
        "key": {
          "type": "UInt64",
          "value": "2006"
        },
        "value": {
          "type": "UInt64",
          "value": "2837670"
        }
      },
      {
        "key": {
          "type": "UInt64",
          "value": "2020"
        },
        "value": {
          "type": "UInt64",
          "value": "808"
        }
      },
      {
        "key": {
          "type": "UInt64",
          "value": "2026"
        },
        "value": {
          "type": "UInt64",
          "value": "765"
        }
      },
      {
        "key": {
          "type": "UInt64",
          "value": "2037"
        },
        "value": {
          "type": "UInt64",
          "value": "13"
        }
      }
    ]
  }
]
```
- **Verification**: https://github.com/onflow/flow-core-contracts/blob/1655a34a1ff5b9a3272b7bcdd791101bd0af1e52/transactions/FlowServiceAccount/scripts/get_execution_effort_weights.cdc

The last entry in the dictionary argument represents the new execution effort weight for the EVM gas (key: 2037, weight: 13). The rest of the arguments are the other weights and are the same as currently set on mainnet.
